lib/spack/spack/test/flag_handlers.py

```diff
@@ -46,14 +56,14 @@ def add_o3_to_build_system_cflags(pkg, name, flags):
 
 @pytest.mark.usefixtures('config')
```

Comment:
Pedantic comment: if all tests use `mock_packages` (a.k.a. `mock_repo`) you can add it to this line and avoid modifying the signature below.